draft-iab-protocol-maintenance.md

```diff
@@ -397,7 +397,8 @@ faulty but does not define how to handle that input, the implementer can provide
 significant value to the ecosystem by reporting the issue and helping evolve the
 specification.
 
-Maintaining specifications so that they closely match deployments ensures that
+Maintaining specifications and all implementations
+so that they closely match all deployments ensures that
```

Comment:
```suggestion
Maintaining specifications and implementations
so that they closely match deployments ensures that
```

There is only the Internet, so "all" is unnecessary.  It also sets up the message of the draft as an impossibility, rather than an aspiration.